pkg/registry/file/dynamicpathdetector/tests/benchamark_test.go

```diff
@@ -0,0 +1,116 @@
+package dynamicpathdetectortests
```

Comment:
typo in the file name `benchmark_test.go`

Reply:
DONE